Makefile

```diff
@@ -41,7 +41,7 @@ DEFAULTORG ?= calico
 DEFAULTIMAGE ?= $(DEFAULTORG)/bpftool:$(VERSION)
 ARCHIMAGE ?= $(DEFAULTIMAGE)-$(ARCH)
 BPFTOOLIMAGE ?= $(DEFAULTIMAGE)-$(BUILDARCH)
-KERNELREF ?= master
+KERNELREF ?= $(VERSION)
```

Comment:
Should version default to a valid kernel version?

Reply:
ok, let's set it to v5.0

Reply:
good catch :)